datasets/us/ice_wanted/crawler.py

```diff
@@ -0,0 +1,153 @@
+from zavod import Context
+from xml.etree import ElementTree
+from normality import collapse_spaces
+
+
+def get_element_text(doc: ElementTree, xpath_value: str, to_remove=[]) -> str:
+    """Extract text from each child nodes of an xpath and joins them together
+
+    Args:
+        doc (ElementTree): HTML Tree
+        xpath_value (str):  xpath to extract text from
+        to_remove (list, optional): string to remove in the extracted text.
+    """
+    element_tags = doc.xpath(xpath_value)
+
+    tag_list = []
+    for tag in element_tags:
+        tag_list.append(tag.text_content())
+
+    element_text = "".join(tag_list)
```

Comment:
let's join on `" "` so we don't inadvertently get something like `end of something.New something` - or is that prevented in some other way?